drivers/serial/Kconfig.gecko

```diff
@@ -5,8 +5,8 @@
 
 config UART_GECKO
 	bool "Gecko UART/USART driver"
-	depends on HAS_SILABS_GECKO
-	depends on GPIO_GECKO
+	default y
+	depends on DT_HAS_SILABS_GECKO_USART_ENABLED
```

Comment:
I believe this should be enabled for DT_HAS_SILABS_GECKO_USART_ENABLED as well as DT_HAS_SILABS_GECKO_UART_ENABLED